modules/simple-payments/paypal-express-checkout.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This PaypalExpressCheckout global is included by wp_enqueue_script( 'paypal-express-checkout' );
+ * It handles communication with Paypal Express checkout and public-api.wordpress.com for the purposes
+ * of simple-payments module.
+ */
+
+/* global paypal */
+/* exported PaypalExpressCheckout */
+/* jshint unused:false */
+var PaypalExpressCheckout = {
+	getCreatePaymentEndpoint: function( blogId ) {
+		return 'https://public-api.wordpress.com/wpcom/v2/sites/' + blogId + '/simple-payments/paypal/payment';
+	},
+	getExecutePaymentEndpoint: function( blogId, paymentId ) {
+		return 'https://public-api.wordpress.com/wpcom/v2/sites/' + blogId + '/simple-payments/paypal/' + paymentId + '/execute';
+	},
+	getNumberOfItems( field, enableMultiple ) {
+		var numberField, number;
+		if ( enableMultiple !== '1' ) {
+			return 1;
+		}
+		numberField = document.getElementById( field );
+
+		if ( ! numberField ) {
+			return 1;
+		}
+		number = Number( numberField.value );
```

Comment:
`parseInt` returns `NaN` as well in some cases:

> An integer number parsed from the given string. If the first character cannot be converted to a number, NaN is returned. (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt)

But yeah, if `parseInt( '12ab' )` still produces a number, I would consider that a bad behavior so let's stick with `Number()`.